common/src/gas_provider/internal.rs

```diff
@@ -744,10 +744,40 @@ where
         )
     }
 
+    fn create_provision(
```

Comment:
```suggestion
    fn create_supply(
```
Why don't use standard words?

Reply:
discussed. name would be create_deposit because gas tree has no exact chain with messages and replies etc